lib/services/workflow-api-service.js

```diff
@@ -237,7 +237,39 @@ function workflowApiServiceFactory(
     WorkflowApiService.prototype.getWorkflowsByNodeId = function(id, query) {
         var nodeId = ({node: id});
         var mergedQuery = _.merge({}, nodeId, query);
-        return waterline.graphobjects.find(mergedQuery);
+        return waterline.graphobjects.find(mergedQuery).then(
+            function sanitize(workflow) {
```

Comment:
I think it would be nice to define the sanitize function within the model code in on-core rather than here.